contributing/code/bc.rst

```diff
@@ -310,6 +314,10 @@ Change return type                                  Yes             Yes
 **Static Methods**
 Turn non static into static                         No              No
 Turn static into non static                         No              No
+**Constants**
+Add constant                                        Yes             Yes
+Remove constant                                     No              No
+Change value of a constant                          Yes             Yes
```

Comment:
Then, imo, it doesn't make sense to have the event name in a constant. `Symfony\Component\HttpKernel\KernelEvents::CONTROLLER` vs `"kernel.controller"`.